docker-compose.yml

```diff
@@ -46,3 +45,17 @@ services:
         depends_on:
             - database
             - herdbook-frontend-devel
+    r-api:
+         container_name: r-api 
+         build:
+             context: ./
+             dockerfile: .docker/r-api
+         environment:
+            - OPTISEL_PORT=31113
+            - OPTISEL_BIND_ADDR=0.0.0.0
+            - POSTGRES_HOST=herdbook-db
+         volumes:
+            - ./R:/usr/local/optisel/R
+         image: r-api
+         ports:
+             - 31113:31113 
```

Comment:
It's fine to have the port open for debugging, but you don't need to as the containers can connect to each other anyway!

Reply:
Yes copy paste from my side it should not be open!